scripts/export_large_tsv/export_large_tsv_python3.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+"""Download a remote tsv from a Terra workspace data model when it is too large to export from Terra UI."""
+from firecloud import api as fapi
+from tqdm import tqdm
+import argparse
+import math
+
+
+def download_tsv_from_workspace(project, workspace, entity, tsv_name, page_size):
+    """Download large TSV file from Terra workspace by designated number of rows."""
+    # get all entity types in workspace using API call
+    # API = https://api.firecloud.org/#!/Entities/getEntityTypes
+    request = fapi.list_entity_types(project, workspace)
+    if request.status_code != 200:
+        print(request.text)
```

Comment:
probably want to add a `break` or something after this - you don't want to continue with the code if you don't get a 200 response